sentry.client.config.js

```diff
@@ -0,0 +1,32 @@
+// This file configures the initialization of Sentry on the client.
+// The config you add here will be used whenever a users loads a page in their browser.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "https://1b2decb65870413bb10959f3eb1d227b@o4505546397712384.ingest.sentry.io/4505546453811200",
```

Comment:
We are using this in two places. I think its best if this goes to `next.config.js` `env` section.